source/util/SGM/SGMSettings.cpp

```diff
@@ -44,7 +44,7 @@ void SGMSettings::load(){
 #ifdef Q_WS_MAC
 	SGMDataFolder_ = settings.value("SGMDataFolder", "/Users/fawkes/Documents/CLS/SGM/ACQUAMANDATA").toString();
 #else
-	SGMDataFolder_ = settings.value("SGMDataFolder", "/home/sgm/beamline/programming").toString();
+	SGMDataFolder_ = settings.value("SGMDataFolder", "/home/workmai/beamline/programming").toString();
```

Comment:
Is this okay that it's only tied to your account or do we have to worry about this at a later date since you've replaced the main sgm account in the path?